src/demomod/README

```diff
@@ -1,3 +1,5 @@
+SPDX-FileCopyrightText: © Fossology contributors
+
 SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Maybe try using the comment format for Markdown. Found some information [here](https://stackoverflow.com/a/32190021).

Reply:
Corrected.